bindings/wasm/manifold-encapsulated-types.d.ts

```diff
@@ -507,6 +516,9 @@ export class Manifold {
    * @param v The vector to add to every vertex.
    */
   translate(v: Vec3): Manifold;
+  translate(x: number): Manifold;
+  translate(x: number, y: number): Manifold;
+  translate(x: number, y: number, z: number): Manifold;
```

Comment:
If this is really what we want, it can be condensed to `translate(x: number, y = 0, z = 0): Manifold;`

Reply:
Ah right didn't think of that. I'll condense it. As mentioned on the other comment, since we are allowed to give these inputs by the implementation, we might as well have it represented in the typescript interface.

Reply:
Condensed to `translate(x: number, y?: number, z?:number)` which does the trick of not warning on these inputs (which are valid for the implementation).